### PR TITLE
CLYPD-44256 Pass request body struct to create tag method

### PR DESCRIPTION
**Pull request by sushil88**

### CODE REVIEW

File: api_response_test.go [hunk 1]
```diff
@@ -1497,3 +1497,42 @@ func TestGetActivitiesResponse(t *testing.T) {
...
+					},
+				},
+			},
+			want: Tag{
+				DisplayID: "123",
+			},
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
```
Comment (sumits) [inline api_response_test.go]: am curious to know why does this package use table tests if there is only one test case

**Review by sumits — APPROVED** [review]
one ❓

File: api_response_test.go (hunk 1 above)
Comment (sushil88) [reply api_response_test.go]: No specific reason. Just kept the IDE-generated code as it is. (lazy me)

Comment (sushil88) [reply api_response_test.go]: This is done. 4a3f5486d9fc32e01c943c2a321c46274438bcdd

**Review by deoakshay — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**